client/components/pie-chart/index.js

```diff
@@ -97,15 +102,16 @@ class PieChart extends Component {
 					preserveAspectRatio={ 'xMidYMid meet' }
 				>
 					<g transform={ `translate(${ SVG_SIZE / 2 }, ${ SVG_SIZE / 2 })` }>
-						{ dataTotal > 0 ? this.renderPieChart() : this.renderEmptyChart() }
-					</g>
+						{' '}
+						{ dataTotal > 0 ? this.renderPieChart() : this.renderEmptyChart() }{' '}
+					</g>{' '}
```

Comment:
Where are these extra spaces coming from? Are they needed?

Reply:
I am not very sure, but looks like my code editor caused these changes as well.